frontend/src/router/pages/ReviewChanges.scss

```diff
@@ -0,0 +1,47 @@
+.page.review-changes {
+  .sub-area {
+    margin-bottom: var(--layout-margin-large);
+    padding-block: var(--layout-padding-medium);
+
+    & > h2 {
+      margin-bottom: var(--layout-margin-xlarge);
+      display: flex;
+      gap: var(--layout-margin-medium);
+
+      .sub-area-icon {
+        height: 1.25em;
+      }
+    }
+  }
+
+  .date-range {
+    display: flex;
+    justify-content: space-between;
+    width: fit-content;
+
+    .date {
+      min-width: 6em;
+      max-width: fit-content;
+    }
+
+    .separator {
+      margin-inline: 1em;
+    }
+  }
+
+  // spacing between date range values
+  td:not(:last-child) .date-range {
+    margin-right: 1em;
+  }
+
+  .notes {
+    .note-metadata {
+      color: var(--typography-color-secondary);
+      filter: brightness(1.5); // muted font color, lighter than color-secondary
+    }
```

Comment:
Very cool, I didn't know this 😮

Reply:
Ideally there would be a color in the design tokens for this, but the "primary" and "secondary" font colors look pretty much the same... Another thing I'll mention to Amanda when we go through things :P I think a custom shade of grey is probably fine to have for this.

SCSS has more color functions, but you'd need the colors to be SCSS variables to use them!
https://sass-lang.com/documentation/modules/color/